tools/sgcollect_info

```diff
@@ -454,6 +454,11 @@ def main():
 
     # Collect the SG binary
     sg_binary_path, _ = get_paths_from_expvars(sg_url)
+    if options.sync_gateway_executable_path is not None and len(options.sync_gateway_executable_path) > 0:
+        if not os.path.exists(sync_gateway_executable_path):
```

Comment:
if options.sync_gateway_executable_path is None or has zero length, can you print a log warning of some sort like "No sync_gateway_executable_path specified, so not collecting sync gateway binary"?  I think that will help debug the cases when the zip is missing an sg binary and we are wondering why ..